node/node.go

```diff
@@ -543,8 +543,7 @@ func (node *Node) Start(ctx context.Context) error {
 		// Start up 'hello' handshake service
 		syncCallBack := func(pid peer.ID, cids []cid.Cid, height uint64) {
 			cidSet := types.NewTipSetKey(cids...)
-			err := node.Syncer.HandleNewTipset(context.Background(), cidSet, pid)
-			if err != nil {
+			if err := node.Syncer.HandleNewTipset(context.Background(), chain.NewPeerHeadInfo(cidSet, types.Uint64(height), pid, true)); err != nil {
```

Comment:
Add an explicit named variable (if the linter will let you) and prominent comment about this innocuous boolean stating that this indicates we are trusting any node, and this should be replaced with [link to issue about bootstrapping].